.github/workflows/end2end-test.yml

```diff
@@ -41,15 +41,41 @@ jobs:
               run: |
                   npx playwright install chromium firefox webkit --with-deps
 
-            - name: Install WordPress and start the server
+            - name: Checkout Playground repository
+              uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+              with:
+                  repository: WordPress/wordpress-playground
+                  path: playground
+
+            - name: Setup Playground
+              run: |
+                  # Install Bun
+                  curl -fsSL https://bun.sh/install | bash # install bun
+
+                  # Install the dependencies
+                  cd playground
+                  npm install
```

Comment:
I know this keeps the Playground updated, but the build artifact from the Playground should be an executable you can download and run directly. That would surely skip a lot of runtime and build steps.

Reply:
Would that simply be https://www.npmjs.com/package/@wp-playground/cli?